main.js

```diff
@@ -56,7 +179,76 @@ function createEachProduct(products) {
 
 function renderPhotos(products) {
   for (var i = 0; i < products.length; i++) {
-    var $product = createEachProduct(products[i])
+    var $product = createEachProduct(products[i], i)
+    products.textContent = products[i].id.value
```

Comment:
The products[i].id.value will always be `undefined`, id doesn't have an `id` property.